server/mvc/models/User.js

```diff
@@ -8,29 +8,21 @@ var config,
 
 module.exports = function(){
   var self = this,
-    document = new ThinkyModel({}),
-    exists = false;
+    document = new ThinkyModel({});
 
   this.findByUserId = function(userId){
```

Comment:
To me, "get" implies an immediate return of the data expected from reading the respective function's name. E.g.:
```javascript
function getBarAsString() {
  return "bar";
}
```
In this case, a promise is returned, which I would not expect or understand if I found any "get" function in some other context, e.g. in a controller.

I understand your point and partly agree. Renaming this function is totally fine with me. I just don't think it should be a "getter".